core-bundle/src/Resources/contao/library/Contao/DcaExtractor.php

```diff
@@ -418,7 +418,7 @@ protected function createExtract()
 
 					if (isset($config['foreignKey']))
 					{
-						$table = substr($config['foreignKey'], 0, strrpos($config['foreignKey'], '.'));
+						$table = substr($config['foreignKey'], 0, strpos($config['foreignKey'], '.'));
```

Comment:
I did not notice any bug with this, however, if you, for example, use the following DCA:

```
// contao/dca/tl_comments.php
$GLOBALS['TL_DCA']['tl_comments']['fields']['member']['foreignKey'] = 'tl_member.CONCAT(firstname,".",lastname)'
```

then the `DcaExtractor` would create the following relation:

```
  "member" => array:4 [
    "table" => "tl_member.CONCAT(firstname,""
    "field" => "id"
    "type" => "belongsTo"
    "load" => "lazy"
  ]
```

which seems wrong of course. As with the other places, the _first_ occurrence of a `.` should split the string.